Orange/widgets/data/owtransform.py

```diff
@@ -51,11 +58,11 @@ def set_input_label_text(self):
 
     def set_template_label_text(self):
         text = "No template data on input."
-        if self.data and self.template_domain is not None:
+        if self.data and self.template_data is not None:
```

Comment:
If `self.template_data` is not None, it evaluates to True (the `is not None` part is not necessary).